### DISCUSSION TITLE
Delete all indexes endpoint

### DISCUSSION THREAD

**Discussion by mmachatschek** [meilisearch/product#608] [Feedback & Feature Proposal]
Hi there,

I propose a new endpoint `DELETE /indexes` which will delete all indexes of a meilisearch instance.

This is especially useful for testing purposes where you want to cleanup the meilisearch instance after/before each test.

Currently you can only delete one index at a time and we would need to wait for each of the tasks to finish. Having a single delete task would be great.

The meilisearch-php SDK had a method called (`deleteAllIndexes()`) [[1]](https://github.com/meilisearch/meilisearch-php/pull/454) which tried to delete all indexes in meilisearch but also had the same issues pointed out here [[2]](https://github.com/laravel/scout/pull/678#discussion_r1099824313).

[[1] https://github.com/meilisearch/meilisearch-php/pull/454](https://github.com/meilisearch/meilisearch-php/pull/454)
[[2] https://github.com/laravel/scout/pull/678#discussion_r1099824313](https://github.com/laravel/scout/pull/678#discussion_r1099824313)

ping @brunoocasali

**Comment by macraig** [comment]
Hi @mmachatschek , apologies for the delayed reply. Thank you for your feedback, we will be considering your feature request in our future planning cycles!
